### PR TITLE
Tx: move and rename constructors

### PR DESCRIPTION
**Pull request by jochem-brouwer**
Part of https://github.com/ethereumjs/ethereumjs-monorepo/issues/3487

Related: https://github.com/ethereumjs/ethereumjs-monorepo/pull/3523

### CODE REVIEW

File: packages/tx/src/1559/constructors.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+ * Notes:
+ * - `chainId` will be set automatically if not provided
+ * - All parameters are optional and have some basic default values
+ */
+export function create1559FeeMarketTx(txData: TxData, opts: TxOptions = {}) {
+  return new FeeMarketEIP1559Transaction(txData, opts)
+}
+
+/**
+ * Create a transaction from a values array.
```
Comment (acolytec3) [inline packages/tx/src/1559/constructors.ts:27]: If we're gonna rename the constructor to `*fromBytesArray`, we should be consistent and update the docs too.  Might be better do something like:
```suggestion
 * Create a transaction from a array of byte encoded values ordered according to the devp2p network encoding - format noted below.
```

File: packages/tx/src/4844/constructors.ts [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+
+  for (let x = 0; x < blobVersionedHashes.length; x++) {
+    const computedVersionedHash = computeVersionedHash(commitments[x], version)
+    if (!equalsBytes(computedVersionedHash, blobVersionedHashes[x])) {
+      throw new Error(`commitment for blob at index ${x} does not match versionedHash`)
+    }
+  }
+}
+
+export function create4844BlobTx(txData: TxData, opts?: TxOptions) {
```
Comment (acolytec3) [inline packages/tx/src/4844/constructors.ts:61]: Missing code docs
```suggestion
/**
 * Instantiate a transaction from a data dictionary.
 *
 * Format: { chainId, nonce, gasPrice, gasLimit, to, value, data, accessList,
 * v, r, s, blobs, kzgCommitments, blobVersionedHashes, kzgProofs }
 *
 * Notes:
 * - `chainId` will be set automatically if not provided
 * - All parameters are optional and have some basic default values
 * - `blobs` cannot be supplied as well as `kzgCommittments`, `blobVersionedHashes`, `kzgProofs`
 * - If `blobs` is passed in,  `kzgCommittments`, `blobVersionedHashes`, `kzgProofs` will be derived by the constructor
 */
export function create4844BlobTx(txData: TxData, opts?: TxOptions) {
```

File: packages/tx/src/4844/constructors.ts [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+      txData.blobs as Uint8Array[],
+      txData.kzgCommitments as Uint8Array[]
+    )
+  }
+
+  return new BlobEIP4844Transaction(txData, opts)
+}
+
+/**
+ * Create a transaction from a values array.
```
Comment (acolytec3) [inline packages/tx/src/4844/constructors.ts:95]: same comment as eip1559

**Comment by jochem-brouwer** [comment]
@acolytec3 this commit should address your review: https://github.com/ethereumjs/ethereumjs-monorepo/pull/3533/commits/4f8d011861164af2c6cb082cf4a124735a159b33

**Review by acolytec3 — APPROVED** [review]

**Review by acolytec3 — APPROVED** [review]
LGTM!

**Pull request merged**